Solopov/DZ3-4_QuizBasicFunctional.SessionAndFilter/ServletTestList/src/main/java/ui/Account/LogoutServlet.java

```diff
@@ -0,0 +1,24 @@
+package ui.Account;
+
+import javax.servlet.ServletException;
+import javax.servlet.http.HttpServlet;
+import javax.servlet.http.HttpServletRequest;
+import javax.servlet.http.HttpServletResponse;
+import javax.servlet.http.HttpSession;
+import java.io.IOException;
+
+public class LogoutServlet extends HttpServlet {
+    @Override
+    protected void doGet(HttpServletRequest req, HttpServletResponse resp) throws ServletException, IOException {
+        HttpSession session=req.getSession();
+        session.removeAttribute("login");
+        req.getRequestDispatcher("Authorization.jsp").forward(req,resp);
+
+
+    }
+
+    @Override
+    protected void doPost(HttpServletRequest req, HttpServletResponse resp) throws ServletException, IOException {
+        doGet(req,resp);
```

Comment:
Why do you need this redirection from post to get?

Reply:
@dinamio Since my servlet is simple and does not handle anything else, I've redefined it as reinsurance so that the code worked anyway. I also read that there is a practice of overriding doGet () and doPost () in the service () method, but I consider this approach redundant because the code is piled up with additional lines.
Maybe I'm wrong about something.I will be glad to hear your opinion on this matter